no-littering.el

```diff
@@ -85,27 +85,27 @@ This variable has to be set before `no-littering' is loaded.")
 
 ;;; Built-in packages
 
-    (setq auto-save-list-file-prefix       (var "saves-"))
-    (setq backup-directory-alist           (list (cons "." (var "backups/"))))
-    (setq bookmark-default-file            (var "bookmarks.el"))
+    (setq auto-save-list-file-prefix       (var "auto-save-"))
+    (setq backup-directory-alist           (list (cons "." (var "backup/"))))
+    (setq bookmark-default-file            (var "bookmark-default.el"))
     (setq desktop-path                     (list (var "desktop/")))
-    (setq desktop-base-file-name           (var "default.el"))
-    (setq desktop-base-lock-name           (var "default.lock"))
+    (setq desktop-base-file-name           (var "desktop-base.el"))
+    (setq desktop-base-lock-name           (var "desktop-base.lock"))
     (setq eshell-directory-name            (var "eshell/"))
-    (setq ido-save-directory-list-file     (var "ido.last"))
-    (setq nsm-settings-file                (var "network-security.data"))
+    (setq ido-save-directory-list-file     (var "ido-save-directory-list"))
+    (setq nsm-settings-file                (var "nsm-settings"))
     (setq org-id-locations-file            (var "org/org-id-locations"))
     (setq org-registry-file                (var "org/org-registry.el"))
-    (setq recentf-save-file                (var "recentf.el"))
-    (setq save-place-file                  (var "saveplace.el"))
+    (setq recentf-save-file                (var "recentf-save.el"))
+    (setq save-place-file                  (var "save-place.el"))
     (setq savehist-file                    (var "savehist.el"))
-    (setq tramp-persistency-file-name      (var "tramp.el"))
+    (setq tramp-persistency-file-name      (var "tramp-persistency.el"))
     (setq trash-directory                  (var "trash/"))
-    (setq url-configuration-directory      (var "url/"))
+    (setq url-configuration-directory      (var "url-configuration/"))
```

Comment:
> This will give the files prefixes that correspond to the packages they relate too.

Okay let's do that for regular files, but not for directories. Here we should stick to the package name.